sklearn/model_selection/tests/test_search.py

```diff
@@ -342,9 +342,17 @@ def test_return_train_score_warn():
     y = np.array([0] * 5 + [1] * 5)
     grid = {'C': [1, 2]}
 
-    estimators = [GridSearchCV(LinearSVC(random_state=0), grid, iid=False),
-                  RandomizedSearchCV(LinearSVC(random_state=0), grid,
-                                     n_iter=2, iid=False)]
+    estimators = [
+        GridSearchCV(
+            LinearSVC(random_state=0, max_iter=100000),
```

Comment:
I would rather leave max_iter to its default. Can you try using `@ignore_warnings(category=ConvergenceWarning))` on test_return_train_score_warn or something like `assert_no_warnings(ignore_warnings(estimator.fit, category=ConvergenceWarning)(...))`

Reply:
Thank's for the review. It's done